defs.h

```diff
@@ -31,10 +31,16 @@
 #if HAVE_CONFIG_H
 #include "config.h"
 #endif
-#if HAVE_LIBZ
-#include <zlib.h>                 // doesn't really belong here, but is missing elsewhere.
-#elif !ZLIB_INHIBITED
-#include "zlib.h"                 // doesn't really belong here, but is missing elsewhere.
+#ifdef ZLIB_H_SYSTEM_INCLUDE
```

Comment:
I don't think so.  The build system may have both the system header and a copy in our source tree, and I want to respect the choice of the user instead of automatically deciding for them.